src/dune/dune.py

```diff
@@ -221,15 +221,28 @@ def state_list(self):
     # pylint: disable=too-many-branches
     def list_nodes(self, simple=False, sep='|'):
 
+        buffer = 3
+        node_name = "Node Name"
+
+        states=self.state_list()
+        name_width = len(node_name) + buffer
+        if not simple:
+            for state in states:
+                name_width = max( len(state.name) + buffer, name_width)
+
         if simple:
             print("Node|Active|Running|HTTP|P2P|SHiP")
         else:
-            print("Node Name\t\t | Active? | Running? | HTTP           | P2P          | SHiP\n"
-                  "----------------------------------------------------------------------------------------------")
+            while len(node_name) < name_width:
+                node_name += " "
+            dash = ""
+            while len(dash) < name_width:
+                dash += "-"
+            print(node_name + "| Active? | Running? | HTTP           | P2P          | SHiP\n"
+                  + dash +    "---------------------------------------------------------------------")
```

Comment:
What do you think about using f-string:
```suggestion
            header = '| Active? | Running? | HTTP           | P2P          | SHiP          '
            print(f'{node_name : <{name_width}}{header}')
            print(f'{"":{"-"}<{name_width + len(header)}}')
```